docs/docs/reference/metaprogramming/inline.md

```diff
@@ -241,18 +241,20 @@ class B extends A {
 }
 
 transparent inline def choose(b: Boolean): A =
-  if b then A() else B
+  if b then A else B
```

Comment:
When calling a no-argument class constructor without using new, the parens are mandatory (https://dotty.epfl.ch/docs/reference/other-new-features/creator-applications.html), so I think in fact this needs to be:
```suggestion
  if b then A() else B()
```
... though perhaps a different example should be used which doesn't use an unrelated new feature people might be unfamiliar with.

Reply:
Agreed, I am going to change it to `new A() ...`